mm-bot/starknet.py

```diff
@@ -93,11 +104,15 @@ async def withdraw(order_id, block, slot) -> bool:
         selector=get_selector_from_name("withdraw"),
         calldata=[order_id, 0, block, 0, slot_low, slot_high]
     )
-    try:
-        transaction = await account.sign_invoke_transaction(call, max_fee=10000000000000)
-        result = await account.client.send_transaction(transaction)
-        await account.client.wait_for_tx(result.transaction_hash)
-
-        logger.info(f"[+] Withdrawn from starknet: {hex(result.transaction_hash)}")
-    except Exception as e:
-        logger.error(f"[-] Failed to withdraw from starknet: {e}")
+    for account in accounts:
+        try:
+            transaction = await account.sign_invoke_transaction(call, max_fee=10000000000000)
```

Comment:
We will need to be really careful with max_fee in the future. Gas fee administration is going to be really important in both ethereum and starknet.
We should add this as a TODO conversation for the future.

Reply:
I added a TODO